src/components/emailSelector/Autocomplete.tsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import cx from 'classnames';
 import uiUtilities from '@teamleader/ui-utilities';
 
 import theme from './theme.css';
 import Box from '../box';
 import { Heading4 } from '../typography';
 import EmailSuggestion from './EmailSuggestion';
+import { Suggestion } from './types';
 
-const Autocomplete = ({ suggestions, highlightedSuggestion, onClick, onHover, renderSuggestion }) => {
+interface AutocompleteProps {
+  suggestions: Suggestion[] | { [x: string]: Suggestion[] };
+  highlightedSuggestion?: Suggestion | null;
+  onClick?: (suggestion: Suggestion, event: React.SyntheticEvent) => void;
+  onHover?: (suggestion: Suggestion, event: React.SyntheticEvent) => void;
```

Comment:
These should be required

Reply:
They are used as `onMouseOver={onHover?.bind(null, suggestion)}` so they seem optional to me?